src/Redux/Store/Randomizer/BasicNemesisCards/RandomCards/reducerHelpers/helpers.ts

```diff
@@ -86,17 +88,17 @@ export const drawPowersAndAttacks = (
 export const getRandomBasicNemesisCardsByTier = (
   cards: ReadonlyArray<types.BasicNemesisCard>,
   tier: types.NemesisCardTier,
-  playerCount: PlayerCount,
+  playerCount: types.PlayerCount,
   seed?: types.Seed
 ) => {
   // Get cards by tier
   const cardsByTier = getBasicNemesisCardsByTier(cards, tier)
 
   // Get amount of minions per player
   const minionAmount =
-    playerCount === 1
-      ? getRandomMinionAmount(2, seed)
-      : getRandomMinionAmount(3, seed)
+    playerCount === 1 && tier === 1
+      ? getRandomMinionAmount(0, 2, seed)
```

Comment:
@jannydiamond it would be preferable if the api of `getRandomMinionAmount` clearer shows its intent. If I understand the code correctly, we draw between 0 and 1 cards for 1 player at tier 1 and for everything else we draw between 1 and 2 cards. However your count is exclusive -> therefore we call the function with 2 and 3 respectively.

I find that rather unintuitive and would prefer the `getRandomMinionAmount` function to substracts the -1 itself and makes the value inclusive.

Reply:
I will merge this anyway - its just to share my thoughts on this